public/javascripts/Admin/src/Admin.js

```diff
@@ -768,44 +768,66 @@ function Admin(_, $, c3, turf, difficultRegionIds) {
+                              allTimes.push({time: turkerData[i].time, binned: Math.min(200.0, turkerData[i].time)});
+                              turkerTimes.push({time: turkerData[i].time, binned: Math.min(200.0, turkerData[i].time)});
+                          }
+
+                          var allStats = getSummaryStats(allTimes, "time");
+                          var regStats = getSummaryStats(regTimes, "time");
+                          var anonStats = getSummaryStats(anonTimes, "time");
+                          var turkerStats = getSummaryStats(turkerTimes, "time");
+
+                          var allHistOpts = {
+                              col: "binned", xAxisTitle: "Total Auditing Time (minutes) - All Users",
+                              yAxisTitle: "Counts (users)", xDomain: [0, 200], width: 187, height: 250,
+                              binStep: 10, legendOffset: -80
+                          };
+                          var regHistOpts = {
+                              col: "binned", xAxisTitle: "Total Auditing Time (minutes) - Registered Users",
+                              yAxisTitle: "Counts (users)", xDomain: [0, 200], width: 187, height: 250,
+                              binStep: 10, legendOffset: -80
+                          };
+                          var turkerHistOpts = {
+                              col: "binned", xAxisTitle: "Total Auditing Time (minutes) - Turker Users",
+                              yAxisTitle: "Counts (users)", xDomain: [0, 200], width: 187, height: 250,
+                              binStep: 10, legendOffset: -80
+                          };
+                          var anonHistOpts = {
+                              col: "binned", xAxisTitle: "Total Auditing Time (minutes) - Anon Users",
+                              yAxisTitle: "Counts (users)", xDomain: [0, 200], width: 187, height: 250,
+                              binStep: 10, legendOffset: -80
+                          };
+
+                          var allChart = getVegaLiteHistogram(allTimes, allStats.mean, allStats.median, allHistOpts);
+                          var regChart = getVegaLiteHistogram(regTimes, regStats.mean, regStats.median, regHistOpts);
+                          var turkerChart = getVegaLiteHistogram(turkerTimes, turkerStats.mean, turkerStats.median, turkerHistOpts);
+                          var anonChart = getVegaLiteHistogram(anonTimes, anonStats.mean, anonStats.median, anonHistOpts);
+
+                          $("#all-audittimes-std").html((allStats.std).toFixed(2) + " Minutes");
+                          $("#reg-audittimes-std").html((regStats.std).toFixed(2) + " Minutes");
+                          $("#turker-audittimes-std").html((turkerStats.std).toFixed(2) + " Minutes");
+                          $("#anon-audittimes-std").html((anonStats.std).toFixed(2) + " Minutes");
+
+                          var combinedChart = {"hconcat": [allChart, regChart, turkerChart, anonChart]};
```

Comment:
Very minor, but I would just go with all, turker, registered, anon; seems weird to split up registered and anon users for some reason.